services/core-api/app/api/mines/documents/resources/mine_document_resource.py

```diff
@@ -105,6 +111,7 @@ class MineDocumentArchiveResource(Resource, UserMixin):
     @api.expect(ARCHIVE_MINE_DOCUMENT)
     @api.response(204, 'Successfully archived documents')
     def patch(self, mine_guid):
+        project_guid = request.args.get('project_guid', type=str)
```

Comment:
What happens when we send a document without a project?

Reply:
100%, it would be great if we wouldn't have to pass in the `project_guid` here and instead look up the related project instead

Reply:
Resolved by passing entity_type param from front end.